presentation/src/main/java/com/mashup/alcoholfree/presentation/ui/component/SulSulButton.kt

```diff
@@ -0,0 +1,187 @@
+package com.mashup.alcoholfree.presentation.ui.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mashup.alcoholfree.presentation.R
+import com.mashup.alcoholfree.presentation.ui.component.model.SulSulButtonColor
+import com.mashup.alcoholfree.presentation.ui.component.model.SulSulButtonSize
+import com.mashup.alcoholfree.presentation.ui.theme.White
+
+@Composable
+fun SulSulDefaultButton(
+    modifier: Modifier = Modifier,
+    content: String,
+    buttonColor: SulSulButtonColor,
+    buttonSize: SulSulButtonSize,
+) {
+    Box(
+        modifier = modifier
+            .clip(shape = RoundedCornerShape(buttonSize.radius.dp))
+            .background(color = buttonColor.color),
+    ) {
+        Row(
+            horizontalArrangement = Arrangement.Center,
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = modifier.padding(
+                horizontal = buttonSize.horizontalPadding.dp,
+                vertical = buttonSize.verticalPadding.dp,
+            ),
+        ) {
+            Text(
+                text = content,
+                style = buttonSize.textSize,
+                color = White,
+            )
+        }
+    }
+}
+
+@Composable
+fun SulSulIconStartButton(
+    modifier: Modifier = Modifier,
+    imgSrc: Int,
```

Comment:
```suggestion
    imageResId: Int,
```

Src를 넘기는게 아니라 리소스 id를 넘기는거같은데?!
그리고 웬만하면 `img`같은 축약어를 사용하지 않는걸 권장해 (src, resId 같은 애들 제외..)

Reply:
넹 수정하겠습니다!